wear/src/main/java/io/homeassistant/companion/android/home/HomeActivity.kt

```diff
@@ -86,6 +87,16 @@ class HomeActivity : ComponentActivity(), HomeView {
         }
     }
 
+    override fun onResume() {
+        super.onResume()
+        SensorWorker.start(this)
+    }
+
+    override fun onPause() {
+        super.onPause()
+        SensorWorker.start(this)
```

Comment:
Shouldn't you stop it here?

Reply:
No you dont want to stop it as its a background service, otherwise the sensors wont continue to update in the background.  The reason why the WebviewActivity has this call in `onPause` as well as `onResume` is because it allows the App Importance sensor to update letting the user know when the app is open or not.

Reply:
Thanks for the explanation, never mind then!

Reply:
Incidentally, are there any other places where it would be appropriate to put the `SensorWorker.start()` calls? I see in the mobile app the location and other Google Play Services sensors use it. Thought it was a good place to put it. 

Without the call would account for why the sensors didn't get updated! 🤦‍♂️

Reply:
So we want it once when the app starts and then after that anytime you need to force an update like in the case of app importance we needed to send an update when the app is paused. It depends on the situation really. The manifest intents will trigger an update to the receiver directly but won't start the worker on its schedule which is what we need to do in this activity

Reply:
Ah, so I might need to update the `AndroidManifest.xml` to get `SensorReceiver` to receive specific intents.

Reply:
Only certain intents can be registered in the manifest others will need to be a runtime receiver. What we have in the manifest today is what can be there.